dnf/query.py

```diff
@@ -49,6 +49,15 @@ def downgrades(self):
         # :api
         return self.filter(downgrades=True)
 
+    def duplicated(self, exclude=[]):
```

Comment:
thanks for making it systematic with other methods (returning the query instead of package list)